Jenkinsfile.d/core/package

```diff
@@ -82,7 +82,7 @@ pipeline {
     GPG_FILE                  = 'jenkins-release.gpg'
     GPG_PASSPHRASE            = credentials('release-gpg-passphrase')
     PACKAGING_GIT_REPOSITORY  = 'git@github.com:jenkinsci/packaging.git'
-    PACKAGING_GIT_BRANCH      = 'INFRA-910-core-release-automation'
+    PACKAGING_GIT_BRANCH      = 'master'
```

Comment:
Not stable-2.263?

Reply:
Agreed.  The [README](https://github.com/jenkins-infra/release/tree/stable-2.263#stable) says that it should be set to the appropriate branch.